tox.ini

```diff
@@ -33,6 +33,13 @@ deps = cmd_examples: {[_cmd_examples]deps}
        build_examples: {[_build_examples]deps}
        flakes: {[_flakes]deps}
        all: {[_all]deps}
+       nose
+
+commands = nosetests -vv --nologcapture --with-doctest --with-coverage --cover-package=pyct
+           pyct --help
+           pyct --version
+           pyct report --help           
+           pyct report pyct python which
```

Comment:
I would probably also move these (along with the nosetests command above) to the unit tests section. Even though they are not strictly unit tests, they are quick to run.

The tests on travis are failing because this whole `commands` key is a duplicate at the moment (there's a commands key just above), which tox is complaining about.